.github/actions/setup-nodejs/action.yaml

```
@@ -26,5 +26,5 @@ runs:
     - if: ${{ inputs.prod == 'true' }}
       name: Install prod dependencies
       shell: bash
-      run: pnpm i --prod
+      run: pnpm i --prod --ignore-scripts true
```

```suggestion
      run: pnpm i --prod --ignore-scripts
```

Don't think we need the boolean here